XCUITests/SettingsTest.swift

```diff
@@ -25,11 +25,14 @@ class SettingsTest: BaseTestCase {
         appsettingstableviewcontrollerTableviewTable.staticTexts["Use Compact Tabs"].swipeUp()
         appsettingstableviewcontrollerTableviewTable.staticTexts["Passcode"].swipeUp()
         appsettingstableviewcontrollerTableviewTable.staticTexts["Show Tour"].swipeUp()
+        sleep(2)
```

Comment:
As far as I have seen in my tests it does not make a difference, just loading the settings view the Help element exists so it is always true. Even if you put that line before scrolling. I tried with .isEnabled and .isHittable but it does not make a difference either. The first one is always true if the element is reached and the other does not work. Locally I do not need that sleep, but lets see if having it in bb makes this test more stable at least till finding a better solution (I have not find any other yet :()